topostats/validation.py

```diff
@@ -28,9 +28,10 @@ def validate_config(config: dict):
             "quiet": Or(True, False, error="Invalid value in config for 'quiet', valid values are 'True' or 'False'"),
             "file_ext": Or(
                 ".spm",
+                ".asd",
                 ".jpk",
                 ".ibw",
-                error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk' or '.ibw'",
+                error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk', '.ibw' or '.asd' when `libasd` imported",
```

Comment:
May have been dealt with in the other PR of .asd support but this can now simply read...

```suggestion
                error="Invalid value in config for 'file_ext', valid values are '.spm', '.jpk', '.ibw' or '.asd'.",
```